tests/xbot/common/injection/wpi_factories/TestCommonLibFactory.java

```diff
@@ -27,5 +29,7 @@ public void makeOneOfEverything() {
         clf.createLidarLite(I2C.Port.kOnboard);
         clf.createAdvancedJoystickButton(j, 1);
         clf.createAnalogHIDButton(j, 1, -1, 1);
+        
+        clf.createHeadingModule(pf.createPIDManager("testo", 1, 0, 0));
```

Comment:
`testo`?

Reply:
Hah, just some "foo" kind of variable.  I'll update to classic random variable naming convention. =]